src/manager/node.c

```diff
@@ -901,38 +891,27 @@ static int node_disconnected(UNUSED sd_bus_message *message, void *userdata, UNU
 
                 if (send_state_change) {
                         LIST_FOREACH_SAFE(subs, usub, next_usub, usubs->subs) {
```

Comment:
Yes, good point. I'll change it to use the `node_compute_unique_monitor_list` instead. 
I think we want to have the same behavior for wildcard subs as we have for regular ones. This ensures that the user of the monitor knows when an agent disconnected and it might want to catch up with the latest state if it reconnects.